tests/local_geometry/test_mxh.py

```diff
@@ -215,47 +215,54 @@ def test_grad_r(generate_miller, parameters, expected):
 def test_load_from_eq():
     """Golden answer test"""
 
+    norms = SimulationNormalisation("test_load_from_eq_mxh")
     eq = read_equilibrium(template_dir / "test.geqdsk", "GEQDSK")
 
     mxh = LocalGeometryMXH()
-    mxh.from_global_eq(eq, 0.5)
+    mxh.from_global_eq(eq, 0.5, norms)
 
     assert mxh["local_geometry"] == "MXH"
 
+    units = norms.units
+
     expected = {
-        "B0": 2.197104321877944,
-        "Rmaj": 1.8498509607744338,
-        "a_minor": 1.5000747773827081,
-        "beta_prime": -0.9189081293324618,
-        "bt_ccw": 1,
-        "bunit_over_b0": 3.5719517046086984,
-        "dpressure_drho": -1764954.8121591895,
-        "dpsidr": 1.874010706550275,
-        "Fpsi": 6.096777229999999,
-        "ip_ccw": 1,
-        "pressure": 575341.528,
-        "q": 4.29996157,
-        "r_minor": 1.0272473396800734,
-        "rho": 0.6847974215474699,
-        "shat": 0.7706147138551124,
-        "kappa": 3.0302699173285554,
-        "shift": -0.5766834602024067,
-        "s_kappa": -0.20110564435448555,
-        "dZ0dr": 9.223273885642161e-05,
-        "sn": [0.0, 0.45903873, -0.06941584, 0.00112094],
-        "cn": [-1.07040432e-04, 6.73097121e-05, 7.55332714e-07, 8.19418442e-06],
-        "dsndr": [0.0, 0.32807204, -0.02038408, -0.02555297],
-        "dcndr": [2.32569249e-04, -2.70991934e-04, 3.30192292e-05, 4.42607392e-05],
+        "B0": 2.197104321877944 * units.tesla,
```

Comment:
For another PR, we should maybe look at trying to reduce the duplication across the miller tests

Reply:
Yes a lot are similar!